velox/vector/DecodedVector.h

```diff
@@ -283,10 +265,20 @@ class DecodedVector {
   // complex type.
   const void* data_ = nullptr;
 
-  // Null bitmask. One bit for each T in 'data_'. nullptr f if there
-  // are no nulls.
+  // Null bitmask of the base vector if wrappings didn't add nulls
+  // (hasExtraNulls_ is false). Otherwise, null bitmask of the base vector
+  // combined with null bitmasks in all the wrappings (hasExtraNulls_ is true).
+  // May be nullptr if there are no nulls.
   const uint64_t* nulls_ = nullptr;
 
+  // Nulls bitmask containing null bits of the base vector combined with null
+  // bits in all the wrappings. May be nullptr if there are no nulls or
+  // allNullsInitialized_ is false. Initialized on first access.
+  const uint64_t* allNulls_ = nullptr;
+
+  // Boolean indicating whether allNulls_ have been initialized.
+  bool allNullsInitialized_ = false;
```

Comment:
nit: how about isAllNullsInit_?

Reply:
Coding style requests to not abbreviate. Hence, spelling out words fully.